charts/rookout-hybrid/README.md

```diff
@@ -125,6 +125,7 @@ helm upgrade --install rookout rookout/rookout-hybrid \
 | datastore.serviceAccount.create | bool | `true` | Specifies whether a service account should be created |
 | datastore.serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
 | datastore.tolerations | list | `[]` | [Tolerations] for use with node taints |
+| datastore.enableNewDao | bool | `false` | Enable new SQL connector and new DAO schema
```

Comment:
Should we not also recommend increasing the cpu in that case?

Reply:
@dudiRookout received all the disclaimers. It does not require nothing but it is recommended that the client increase resources generally speaking. This is not a mandatory step